src/components/StatsUpdater/StatsUpdater.tsx

```diff
@@ -42,6 +41,43 @@ function StatsUpdater() {
     if (!last || nodeLoad.probe_timestamp > last.probe_timestamp) {
       load.push(nodeLoad);
     }
+  };
+
+  const updateFs = (nodes: NodeStatusInfo[]) => {
```

Comment:
We keep track of stats here, because we need to get the previous value to calculate iops